<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Белки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ккал</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Жиры</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Углеводы</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1500</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Витамины</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Омега</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> A  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="progressBar_6">
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="progressBar_7">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QProgressBar" name="progressBar_8">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QProgressBar" name="progressBar_9">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QProgressBar" name="progressBar_10">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QProgressBar" name="progressBar_11">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>B7?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QProgressBar" name="progressBar_12">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B9</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QProgressBar" name="progressBar_13">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B12</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QProgressBar" name="progressBar_14">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QProgressBar" name="progressBar_15">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QProgressBar" name="progressBar_16">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QProgressBar" name="progressBar_17">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QProgressBar" name="progressBar_18">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Железо</string>
       </property>
       <property name="text">
        <string>Fe</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="progressBar_19">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_24">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Кальций</string>
       </property>
       <property name="text">
        <string>Ca</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="progressBar_20">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_25">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Йод</string>
       </property>
       <property name="text">
        <string>I ?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QProgressBar" name="progressBar_21">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_26">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Калий</string>
       </property>
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QProgressBar" name="progressBar_22">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_27">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Магний</string>
       </property>
       <property name="text">
        <string>Mg</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QProgressBar" name="progressBar_23">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_28">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Медь</string>
       </property>
       <property name="text">
        <string>Cu</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QProgressBar" name="progressBar_24">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_29">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Фосфор</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QProgressBar" name="progressBar_25">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_30">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Фтор</string>
       </property>
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QProgressBar" name="progressBar_26">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_31">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Цинк</string>
       </property>
       <property name="text">
        <string>Zn</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QProgressBar" name="progressBar_27">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_34">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Селен</string>
       </property>
       <property name="text">
        <string>Se</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QProgressBar" name="progressBar_28">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_35">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Натрий</string>
       </property>
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QProgressBar" name="progressBar_29">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_36">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>Марганец</string>
       </property>
       <property name="text">
        <string>Mn</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QProgressBar" name="progressBar_30">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Минералы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>217</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Рацион           </string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Введите продукт</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>261</width>
      <height>441</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
